macros/src/lib.rs

```diff
@@ -924,11 +924,11 @@ pub fn internp(ts: TokenStream) -> TokenStream {
     let section = format!(".defmt.prim.{}", sym);
 
     quote!(match () {
-        #[cfg(target_arch = "x86_64")]
+        #[cfg(feature = "unstable-test")]
```

Comment:
this (cfg inside `quote!`) seems fine because `internp` is an implementation detail and only used within defmt non-public / non-user-facing code.